src/services/static-data.service.ts

```
@@ -80,6 +85,95 @@
+            case 'Imperial':
+                return Alliance.Imperial;
+            default:
+                return undefined;
+        }
+    }
+
+    private static convertNpcData(): INpcData[] {
+        let data: INpcData[] = [];
+
+        data = npcData.npcs.map(npc => {
+            const faction: Faction = this.parseFaction(npc.faction) ?? Faction.Ultramarines;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)